wrappers/python/python.cpp

```diff
@@ -397,6 +401,23 @@ PYBIND11_MODULE(NAME, m) {
     py::class_<rs2::points, rs2::frame> points(m, "points");
     points.def(py::init<>())
         .def(py::init<rs2::frame>())
+        .def("get_vertices_2d", [](rs2::points& self) -> BufData
```

Comment:
I think it would be cleaner to have a single `get_vertices` function that takes an int parameter requesting the number of dimensions of the output than `get_vertices` and `get_vertices_2d`.
This function should default to 1 dimension on no input to remain backwards compatible.